eng/pipelines/prepare-pipelines.yml

```
@@ -8,4 +8,4 @@ extends:
     CIConventionOptions: ''
     UPConventionOptions: '--variablegroup 58 --variablegroup 76 --variablegroup 56 --variablegroup 93'
     # Uncomment once live tests (tests.yml) are added to the repository
-    # TestsConventionOptions: '--variablegroup 64'
+    TestsConventionOptions: '--variablegroup 64'
```

Since we're going to do the unified pipeline (live and ci in one), we can move the variable group access into the `UPConventionOptions` (it's also 52, not 64, which I guess I copied over from another repo mistakenly):

```suggestion
    UPConventionOptions: '--variablegroup 58 --variablegroup 76 --variablegroup 56 --variablegroup 93 --variablegroup 52'
```

---

Won't allow me to commit, can you verify the change I made?